frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -24,6 +40,7 @@ const normalizePackageManifests = (packageManifests, kind) => {
     const tags = packageManifest.metadata.tags;
     const version = _.get(packageManifest, 'status.channels[0].currentCSVDesc.version');
     const currentCSVAnnotations = _.get(packageManifest, 'status.channels[0].currentCSVDesc.annotations', {});
+    const enabled = _.includes(enabledPackages, name);
```

Comment:
This should actually be determined by whether or not there is an OLM `Subscription` for the given package, which is implemented in #984.

Reply:
The problem with looking at the subscription is that the user might be redirected to marketplace before the subscription is created. In that case, the tile will not be marked as enabled and the user will be able to try to subscribe again. With the dropdowns, I think it's safe to assume the subscription will be created successfully once the catalogsource it points to is created.

Reply:
But I guess it should also check for subscriptions in case one has been created through the command line right?

Reply:
So we should check for both